Resources/Prototypes/_StarLight/Entities/Mobs/Species/cyclorites.yml

```
@@ -119,6 +119,11 @@
           32:
             sprite: _Starlight/Mobs/Species/Cyclorite/displacement.rsi
             state: neck
+      misc: # Starlight start
+        sizeMaps:
+          32:
+            sprite: _Starlight/Mobs/Species/Cyclorite/displacement.rsi
+            state: neck # Starlight end
```

```suggestion
      misc:
        sizeMaps:
          32:
            sprite: _Starlight/Mobs/Species/Cyclorite/displacement.rsi
            state: neck
```

Starlight in starlight???

---

oh shit whoops. I just added starlight comments to everything, forgot that cyclorites were themselves SL lmao